src/modules/live-execution-tracker/live-execution-tracker.ts

```diff
@@ -109,6 +110,14 @@ export class LiveExecutionTracker {
     this._parentProcessInstanceId = await this._getParentProcessInstanceId();
     this._parentProcessModelId = await this._getParentProcessModelId();
 
+    const hasTaskId: boolean = routeParameters.taskId !== undefined
```

Comment:
```suggestion
    const routeParameterContainTaskId: boolean = routeParameters.taskId !== undefined
```

I find this a much better name for the rule.